compliance_checker/cf/cf.py

```diff
@@ -723,9 +725,9 @@ def check_ancillary_data(self, ds):
             anc_result = Result(BaseCheck.HIGH, name=('§3.4 Ancillary Variables', k))
             msgs = []
 
-            if not isinstance(anc, str):
+            if not isinstance(anc, str) and not isinstance(anc, unicode):
```

Comment:
you can add that test in a utils function and always import it from there... i know its used other places in CC already.